packages/p2p/src/components/advertiser-page/advertiser-page.scss

```diff
@@ -429,7 +433,7 @@
         margin-top: 4rem;
 
         @media (max-height: 525px) {
-            height: 8rem;
+            height: 16rem;
```

Comment:
Icon is too small for height `<=` 525px:
![Screenshot 2022-09-30 at 2 24 22 PM](https://user-images.githubusercontent.com/103016120/193211044-8340a52d-c240-4698-8746-34a0520ee423.png)
Adjusted height:
![Screenshot 2022-09-30 at 3 05 45 PM](https://user-images.githubusercontent.com/103016120/193211147-fd679bcc-d8ff-4455-afb0-b934e9852d6a.png)